app/routes/auth.js

```diff
@@ -71,7 +71,7 @@ router.post('/login', (req, res) => {
       sub: user.id,
     };
 
-    const token = jwt.sign(payload, config.auth.secret);
+    const token = jwt.sign(payload, config.auth.secret, { expiresIn: '60d' });
```

Comment:
What is the reasoning behind such a long-lived JWT?

Reply:
Why not? I don't know, I guess we need to agree on a common expiration time.